### PR TITLE
refactor: update social card design

### LINKED ISSUE #44: feature: Update Open Graph Social Card Designs
There have been updates to the social card design

socialDashCard-linkedin-dashboard-card

[[Project Dashboard](https://www.figma.com/file/jymWgr4kOgSpZYEoFrlgZQ/Design-System?type=design&node-id=1277-2474&t=BSQ0EN4E1Wtyb6Fx-4)]

socialUserCard-linkedin-user-profile-card

[[User Profile Card](https://www.figma.com/file/jymWgr4kOgSpZYEoFrlgZQ/Design-System?type=design&node-id=1277-2365&t=BSQ0EN4E1Wtyb6Fx-4)]

socialUserCard-linkedin-highlight

[[Highlights](https://www.figma.com/file/jymWgr4kOgSpZYEoFrlgZQ/Design-System?type=design&node-id=1278-2505&t=BSQ0EN4E1Wtyb6Fx-4)]

### PR DESCRIPTION
**Pull request by OgDev-01**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [x] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
This PR updates the social card design 

## Related Tickets & Documents
closes #44 

### Insights Page 
image
image

### Users
image
image

### Highlights
image
image

## Added tests?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: src/social-card/templates/insight-card.template.ts [hunk 1]
```diff
@@ -8,11 +8,11 @@ const insightCardTemplate = (pageName: string, contributors: string, repos: stri
     <div tw="">
       <div tw="w-full flex-col flex-nowrap" style="gap: -10px;">
         <h1 tw="font-medium text-72px leading-72px text-zinc-900 tracking-tight" style="width: 100%;">
-          ${pageName}: Insights
+          OpenSauced Insights
         </h1>
+        <p tw="font-medium text-48px tracking-tight"> ${pageName}: Insights </p>
```
Comment (brandonroberts) [inline src/social-card/templates/insight-card.template.ts:13]: I don't think we need `: Insights` here if its in the header above

```suggestion
        <p tw="font-medium text-48px tracking-tight"> ${pageName} </p>
```

**Comment by bdougie** [comment]
I'm thinking we might want to add the copy or previews text

**Comment by isabensusan** [comment]
I think it'd look better if we used bold titles instead of semi-bold, and grey for the subtitles (see screenshot), just to ensure a clear content hierarchy, but honestly it's a small detail and it still looks good the way it is right now.

image

**Review by bdougie — CHANGES_REQUESTED** [review]
I know it didn't make the design, but can we add the highlight copy back to the card (in grey). This was pretty useful and as long as we truncate the characters shown to 300, it should work.

**Comment by OgDev-01** [comment]
a quick question @bdougie, are we replacing the username with the highlights copy or adding the copy below the username on the highlights card?

**Comment by bdougie** [comment]
> a quick question @bdougie, are we replacing the username with the highlights copy or adding the copy below the username on the highlights card?

Replace the user name.

**Comment by OgDev-01** [comment]
Updated copy 
image

cc: @bdougie @brandonroberts

**Review by bdougie — APPROVED** [review]

**Review by jpmcb — APPROVED** [review]
Very nice!! 👏🏼 🔥 
jpmcb

I wonder what's going on with the bottom right of my card. Almost looks like the small green gets cut off early before it curves back up?

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**